codesign/lib/codesign.dart

```diff
@@ -2,6 +2,8 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-int calculate() {
-  return 6 * 7;
-}
+export 'src/globals.dart';
```

Comment:
Are you planning on making this package available to something outside of main.dart? 
I think you should remove this and explicitly import the dart files you need where you need them.

Reply:
umm yeah I am planning to make it visible to tests and other dart files. would this structure be less ideal if I am am trying to build something like https://github.com/flutter/cocoon/pull/1861?

Reply:
I think you have it the suggested way then:
https://dart.dev/guides/libraries/create-library-packages#organizing-a-library-package